.github/workflows/ci.yml

```diff
@@ -14,6 +14,7 @@ jobs:
     runs-on: ${{ matrix.os }}
 
     strategy:
+      fail-fast: false
```

Comment:
What's the reason for changing this?

Reply:
If one job fails in matrix it will force fail other jobs immediately. This lets other jobs to finish.

Reply:
Why do we want them to finish? Doesn't cancelling them save us CPU cycles?

Reply:
You actually increase used CPU cycles in reality. I know it sounds counter-intuitive. Of course there are cases that this will eat more CPU cycles, but in most cases - where changes happen mostly in leaf package - this will save CPU.